install.sh

```diff
@@ -303,16 +303,16 @@ if [ "$1" = "install" ]; then
 
   # ccache
   if [ "$_noccache" != "true" ]; then
-
-    if [ "$_distro" = "Ubuntu" ] || [ "$_distro" = "Debian" ]; then
+    # Todo: deal with generic and paths, maybe just export boths possibilities and not care
```

Comment:
Oh this line I forgot to strip it. It's from my other work on making a generic install work (`make install` and co). Since my commits are super clean. Let's just leave it here and say no one ever saw this useless comment.

Reply:
:frog: